scripts/gen.sh

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env bash
+
+set -e
+
+forge build > /dev/null
+
+cd "${SOLIDITY_SRC_DIR}"
+
+for file in ${CONTRACTS}
+do
```

Comment:
Should you handle the case of more than one contract? If you don't want to handle it, suggest removing the loop entirely. If yes, then the script should be able to handle them.
```suggestion
for file in "${CONTRACTS[@]}"; do
```

Reply:
We  will have  more than one  contract after adding the DAVerifier.